src/libman/Commands/CacheCleanCommand.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.CommandLineUtils;
+using Microsoft.Web.LibraryManager.Contracts;
+
+namespace Microsoft.Web.LibraryManager.Tools.Commands
+{
+    /// <summary>
+    /// Defines a command to allow cleaning the libman cache.
+    /// </summary>
+    internal class CacheCleanCommand : BaseCommand
```

Comment:
We talked about disabling this command. I don't think we should list the cache contents.